server/routes/projects.js

```diff
@@ -91,6 +91,52 @@ router.route('/:projectId').get(async (req, res) => {
 	}
 });
 
+router.route('/:project_id').put(authenticateToken, async (req, res) => {
+	const { user } = req;
+	let { name, description, github, media, technologies, deploymentLink } =
+		req.body;
+	try {
+		name = isValidProjectName(name);
+		description = req.body.description
+			? isValidStr(req.body.description, 'project description')
+			: null;
+		github = req.body.github ? isValidGithub(req.body.github) : null;
+		technologies = isValidTechnologies(technologies);
+		deploymentLink = req.body.deploymentLink
+			? isValidStr(req.body.deploymentLink, 'project deployment link')
+			: null;
+		const projectId = isValidObjectId(req.params.project_id);
+		await projectsData.getProjectById(projectId);
+		const projectObject = {
+			name: name,
+			description,
+			github,
+			media,
+			technologies,
+			deploymentLink,
+		};
+		const project = await projectsData.updateProject(
+			projectObject,
+			projectId,
+			user
+		);
+		res.json({ project: project, message: 'Project udpated successfully' });
+	} catch (e) {
+		sendErrResp(res, e);
+	}
+})
+.delete(authenticateToken,async (req, res) => {
+	try{
+	const { user } = req;
+		const status = await projectsData.removeProject(req.params.project_id,user);
+		if (status) res.json({message: "Project deleted successfully"   });
```

Comment:
response status should be `204`
there is a success codes object in `utils/indes.js`
you can also refer how sabitha has done it for delete comment